tests/integration_tests/test_panther.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from numpy.testing import assert_allclose
+import pytest
+
+import mantid
+from abins.instruments.panther import PantherInstrument
+
+from resolution_functions.instrument import Instrument
+
+
+WAVENUMBER_TO_MEV = 0.12398419843320028
+MEV_TO_WAVENUMBER = 8.06554465
+
+
+@pytest.fixture(scope="module")
+def panther_rf():
+    return Instrument.from_default('PANTHER', 'PANTHER')
+
+
+@pytest.fixture(scope="module", params=np.arange(0, 1209, 20), ids=lambda e: f'ei={e}')
+def panther_abins_plus_rf_abins_resolution_function(panther_rf, request):
+    abins = PantherInstrument()
```

Comment:
Maybe better to name this `abins_instrument` or `abins_panther`; we are already using `abins` as a package name so it's confusing to see stuff like `abins.calculate_sigma`.